File: src/primus/pawel/Exercise_102.java
package primus.pawel;

import java.util.Arrays;

// minimum sum of array
// 7kyu
// https://www.codewars.com/kata/5a523566b3bfa84c2e00010b/train/java
// 2021-05-01 T:20:44:49
class Exercise_102 {

    public static int minSum(int[] passed)
    {
        int result = 0;
        final int[] clonedArr = passed.clone();
        Arrays.sort(clonedArr);
        for (int i = 0; i < clonedArr.length/2; i++) {
            result += clonedArr[i]* clonedArr[clonedArr.length-1-i];
        }
        return result;
    }

    public static void main(String[] args) {
        minSum((new int[]{12,6,10,26,3,24}));
    }


}

File: src/JS/Exercise_102.js

// Sum of two lowest positive integers
// 7 kyu
// https://www.codewars.com/kata/558fc85d8fd1938afb000014/train/javascript
// 2022-06-07 T:07:39:12

function sumTwoSmallestNumbers(numbers) {
    numbers.sort((a, b) => a - b)
    return numbers[0] + numbers[1];
}
console.log(sumTwoSmallestNumbers([15, 28, 4, 2, 43]))